nav/nav_master/nav_master.py

```
@@ -397,7 +388,13 @@ def create_overlay(self) -> NDArrayUint8Type:
 
         # Create a min_index that is the size of the original image, properly offset.
         # This array indicates which model is at the front for each pixel.
-        min_index = self.obs.extract_offset_image(self.closest_model_index, offset)
+        # TODO This is inefficient because we are creating the combined model a second
+        # time. But we may need to do this in the future if NavTechniqueCorrelateAll
+        # ends up being more complicated. This needs to be revisited.
+        if self.combined_model is None:
+            raise ValueError('Combined model is None')
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)